packages/datadog-instrumentations/test/body-parser.spec.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const getPort = require('get-port')
+const dc = require('diagnostics_channel')
+const axios = require('axios')
+const agent = require('../../dd-trace/test/plugins/agent')
+
+withVersions('body-parser', 'body-parser', version => {
+  describe('body parser instrumentation', () => {
+    const bodyParserReadCh = dc.channel('datadog:body-parser:read:finish')
+    let port, server, requestBody
+
+    before(() => {
+      return agent.load(['express', 'body-parser'], { client: false })
+    })
+    before((done) => {
+      const express = require('../../../versions/express').get()
+      const bodyParser = require(`../../../versions/body-parser@${version}`).get()
+      const app = express()
+      app.use(bodyParser.json())
+      app.post('/', (req, res) => {
+        requestBody()
```

Comment:
Does the current `requestBody()` aims to simulate some controller's processing? or it is something related with the actual body of the request?

In case of the first, I would go with another name (`controllerProcess` or something like that)

Reply:
Yeah, you're right the `requestBody` stub is faking some processing in the express middleware. I will rename it.